metadata.json

```diff
@@ -1,16 +1,25 @@
 {
   "name": "puppetlabs-tomcat",
-  "version": "2.1.0",
+  "version": "2.1.3",
```

Comment:
We shouldn't bump the version as part of this PR, or the github url. That is done as part of the release process.

What is the reason of reformatting the dependency hash?